extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -38,6 +38,7 @@ public static function onAfterInitialize( $title, $article, $output, $user, WebR
 		$wgAdDriverForceDirectGptAd = $request->getBool( 'forcedirectgpt', $wgAdDriverForceDirectGptAd );
 		$wgAdDriverForceLiftiumAd = $request->getBool( 'forceliftium', $wgAdDriverForceLiftiumAd );
 		$wgAdDriverForceTurtleAd = $request->getBool( 'forceturtle', $wgAdDriverForceTurtleAd );
+		$wgAdDriverForceMonetizationService = $request->getBool( 'forcemonetizationservice', $wgAdDriverForceMonetizationService );
```

Comment:
Do we need the force at all? I mean, they are separate slots, not competing against other providers, so force doesn't seem to bring any value.